app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::API
```

Comment:
A good practice is to avoid modifying the `ApplicationController` an leave it as clean as it can be, I highly recommend that you create a base API controller like `ApiController` that inherits from `ApplicationController` and in that controller you mount devise